src/include/units/random.h

```diff
@@ -0,0 +1,453 @@
+#pragma once
+
+#include <random>
+#include <functional>
+
+namespace units {
+
+template<typename Quantity>
+struct uniform_int_distribution : public std::uniform_int_distribution<typename Quantity::rep>
```

Comment:
Should we constrain `Q::rep` only to the ones supported by the distribution?

Reply:
Yes, but I was thinking of a way to do this without having to pass in ```Dimension D, UnitOf<D> U``` as template arguments.

Reply:
I was thinking more of:

```cpp
template<Quantity Q>
  requires std::integral<Q::rep>
struct uniform_int_distribution : public std::uniform_int_distribution<typename Quantity::rep>
```